src/sentry/testutils/factories.py

```diff
@@ -1064,13 +1066,13 @@ def create_internal_integration_token(user, install=None, request=None, org=None
             scopes = []
         if install is None:
             assert org
-            sentry_app = Factories.create_sentry_app(
-                name="Integration Token",
-                organization=org,
-                scopes=scopes,
+            sentry_app = Factories.create_internal_integration(
+                "integration token",
```

Comment:
Also `create_internal_integration_token` was not creating an internal integration, this one's my bad cuz I created this method a while ago :O